packages/jwt-verifier/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 2.0.1
+
+### Other
+
+- [#952](https://github.com/okta/okta-oidc-js/pull/952) - Updates configuration-validation dependency of okta-jwt-verifier to 1.0.0
```

Comment:
nit: probably no need to mention `okta-jwt-verifier` since it's the changelog of `jwt-verifier`